xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAE.cpp

```diff
@@ -198,19 +197,19 @@ void CEngineStats::GetSyncInfo(CAESyncInfo& info, CActiveAEStream *stream)
 float CEngineStats::GetCacheTime(CActiveAEStream *stream)
 {
   CSingleLock lock(m_lock);
-  float delay = 0;
+  double delay = 0;
 
   for (auto &str : m_streamStats)
   {
     if (str.m_streamId == stream->m_id)
     {
       CSingleLock lock(stream->m_statsLock);
-      float buffertime = str.m_bufferedTime + stream->m_bufferedTime;
+      double buffertime = str.m_bufferedTime + stream->m_bufferedTime;
       delay += buffertime / str.m_resampleRatio;
       break;
     }
   }
-  return delay;
+  return static_cast<double>(delay);
```

Comment:
Think better change them everywhere to float or double, other *GetCacheTime` return double. Also has this function float as return and the static_cast a bit wrong :-)